webpack.demos.config.js

```diff
@@ -23,5 +26,11 @@ module.exports = {
   },
   eslint: {
     configFile: '.eslintrc'
+  },
+  devServer: {
```

Comment:
hm, we need to shuffle some stuff around if we're to add `devServer`. when running a clean `npm run start:demos`, the outputs from each demo are off by one directory–they're requesting `/[name]/index.js` but the dev server is serving it at `/demos/[name]/index.js`.